src/Migration/Migration1605714035FinSearchConfig.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Migration;
+
+use Doctrine\DBAL\Connection;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+class Migration1605714035FinSearchConfig extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1605714035;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $sql = 'CREATE TABLE IF NOT EXISTS `finsearch_config` (
+  `id` binary(16) NOT NULL,
+  `configuration_key` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `configuration_value` json NOT NULL,
+  `sales_channel_id` binary(16) NOT NULL,
+  `language_id` binary(16) NOT NULL,
+  `created_at` datetime(3) NOT NULL,
+  `updated_at` datetime(3) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `uniq.system_config.configuration_key__sales_channel_id` (`configuration_key`,`sales_channel_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci';
+
+        $connection->executeUpdate($sql);
+
+        $this->insertPreviousConfigurationIfExists($connection);
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
+        // implement update destructive
+    }
+
+    private function insertPreviousConfigurationIfExists(Connection $connection): void
+    {
+        $connection->executeUpdate('TRUNCATE TABLE `finsearch_config`');
+
+        $sql = "SELECT LOWER(HEX(`id`)) AS `id`, 
```

Comment:
Use [heredoc syntax](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) when writing a string in multiple lines.